cmd/clusterctl/examples/tools/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.12
```

Comment:
golang:1.12 is a beefy image, ~700MB. I couldn't find anything smaller than this that has go installed. If we think this is too big for a tools container we could install go ourselves on a slimmer base image, thoughts?

Reply:
Hi @andrewsykim,

Unless we need Go you could use `debian:stretch` as a base and then just build the tools you need in a build phase and copy them into the final phase.

Reply:
Going to use `debian:stretch-slim` unless there are objections

Reply:
That's what I meant anyway, so go for it. FYI, you may wish to set `GOOS`, `GOHOSTOS`, etc. in the container just so anything that depends on those will find them already set.